samples/vsphere/common/vapiconnect.py

```diff
@@ -44,6 +44,8 @@ def connect(host, user, pwd, skip_verification=False, suppress_warning=True):
     session = requests.Session()
     if skip_verification:
         session = create_unverified_session(session, suppress_warning)
+    elif cert_path:
+        session = create_verified_session(session, cert_path, suppress_warning)
```

Comment:
You can just set the session.verify directly without the helper method since you don't need suppress warning any more.